services/web/client/source/class/osparc/desktop/WorkbenchView.js

```diff
@@ -303,7 +303,7 @@ qx.Class.define("osparc.desktop.WorkbenchView", {
         const nodeId = e.getData();
         this.__removeNode(nodeId);
       }, this);
-      this.__sidePanel.addOrReplaceAt(new osparc.desktop.PanelView(this.tr("Service tree"), nodesTree), 0, {
+      this.__sidePanel.addOrReplaceAt(new osparc.desktop.PanelView(this.tr("Nodes tree"), nodesTree), 0, {
```

Comment:
I guess that's what we're calling them now =)

Reply:
And we call "Sharing" to "Permissions"